discovery-provider/src/models/models.py

```diff
@@ -581,6 +581,17 @@ def __repr__(self):
 play_item_id={self.play_item_id},\
 count={self.count}>"
 
+class LatestSlots(Base):
+    __tablename__ = "latest_slots"
+
+    tablename = Column(String, primary_key=True, nullable=False, index=False)
+    slot = Column(Integer, nullable=False, index=False)
```

Comment:
As we add more aggregates to this table, we might use other values like blocks. Would sticking with the slots name be ok?

Reply:
not as particular about these names, but a more generic might be better? like `LatestIndexingState` or something

Reply:
Yeah, that might be better.